build/docker/build.arm64.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euxo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+cd "$DIR/../.."
+
+PUBLISH_OUTPUT="$( pwd )/src/bin/managed-publish"
+mkdir -p "$PUBLISH_OUTPUT/netstandard2.0"
+mkdir -p "$PUBLISH_OUTPUT/netcoreapp3.1"
+
+dotnet build -c $buildConfiguration src/Datadog.Trace.ClrProfiler.Managed.Loader/Datadog.Trace.ClrProfiler.Managed.Loader.csproj
+
+for proj in Datadog.Trace Datadog.Trace.OpenTracing ; do
+    dotnet publish -f netstandard2.0 -c $buildConfiguration src/$proj/$proj.csproj
+    dotnet publish -f netcoreapp3.1 -c $buildConfiguration src/$proj/$proj.csproj
+done
+
+dotnet publish -f netstandard2.0 -c $buildConfiguration src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj -o "$PUBLISH_OUTPUT/netstandard2.0"
+dotnet publish -f netcoreapp3.1 -c $buildConfiguration src/Datadog.Trace.ClrProfiler.Managed/Datadog.Trace.ClrProfiler.Managed.csproj -o "$PUBLISH_OUTPUT/netcoreapp3.1"
+
+# Only build Samples.AspNetCoreMvc31 for netcoreapp3.1
+if [ "$publishTargetFramework" == "netcoreapp3.1" ]
+then
+    dotnet publish -f $publishTargetFramework -c $buildConfiguration test/test-applications/integrations/Samples.AspNetCoreMvc31/Samples.AspNetCoreMvc31.csproj -p:Configuration=$buildConfiguration -p:ManagedProfilerOutputDirectory="$PUBLISH_OUTPUT"
+fi
+
+for sample in Samples.Elasticsearch Samples.Elasticsearch.V5 Samples.ServiceStack.Redis Samples.StackExchange.Redis Samples.SqlServer Samples.Microsoft.Data.SqlClient Samples.MongoDB Samples.HttpMessageHandler Samples.WebRequest Samples.Npgsql Samples.MySql Samples.GraphQL Samples.FakeKudu Samples.Dapper Samples.NoMultiLoader Samples.RabbitMQ Samples.RuntimeMetrics ; do
```

Comment:
The way we manage building these samples for tests is getting out of hand.
Not to fix for this PR, but I think we should work on sharing some of this between scripts.

Reply:
Yeah, I agree but we have to be careful, at least in arm64 I had to disable the orchard based ones because is not compatible with the architecture. Also I had to disable anything related with aspnetcore 2.x